apps/form.go

```diff
@@ -53,3 +56,56 @@ type Form struct {
 	// Fields is the list of fields in the form.
 	Fields []Field `json:"fields,omitempty"`
 }
+
+func (f *Form) UnmarshalJSON(data []byte) error {
+	stringValue := ""
+	err := json.Unmarshal(data, &stringValue)
+	if err == nil {
+		*f = Form{
+			Source: &Call{
+				Path: stringValue,
+			},
+		}
+		return nil
+	}
+
+	// Need a type that is just like Form, but without UnmarshalJSON
+	structValue := struct {
+		Source        *Call   `json:"source,omitempty"`
+		Title         string  `json:"title,omitempty"`
+		Header        string  `json:"header,omitempty"`
+		Footer        string  `json:"footer,omitempty"`
+		Icon          string  `json:"icon,omitempty"`
+		Submit        *Call   `json:"submit,omitempty"`
+		SubmitButtons string  `json:"submit_buttons,omitempty"`
+		Fields        []Field `json:"fields,omitempty"`
+	}{}
```

Comment:
Can we make a linting rule or something similar to make sure this stays in sync?

Reply:
That would be awesome, but not sure how to achieve it, and a bit low on mana to look into it. Are you OK if we create a ticket for it?

Reply:
@larkox Yeah this may be a good HW ticket too

Reply:
Maybe for now we put a comment near the `Form` struct definition saying to make sure to update this struct.

Reply:
done.